aether-common-module/aether/common/conf/settings.py

```diff
@@ -232,6 +232,26 @@
 if os.environ.get('DJANGO_HTTP_X_FORWARDED_PROTO', False):    # pragma: no cover
     SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
+# Storage Configuration
+# ------------------------------------------------------------------------------
+
+CUSTOM_STORAGE = os.environ.get('DJANGO_REMOTE_STORAGE', '')
+
+if CUSTOM_STORAGE:   # pragma: no cover
+    INSTALLED_APPS += ['storages', ]
+    if CUSTOM_STORAGE.lower() == 's3':
+        DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
+        AWS_AUTO_CREATE_BUCKET = True
+        AWS_ACCESS_KEY_ID = os.environ.get('AWS_ACCESS_KEY_ID', '')
+        AWS_SECRET_ACCESS_KEY = os.environ.get('AWS_SECRET_ACCESS_KEY', '')
+        AWS_STORAGE_BUCKET_NAME = os.environ.get('BUCKET_NAME',
+                                                 os.environ.get('HOSTNAME', 'aether-project'))
```

Comment:
I would suggest that we avoid defaults, because if we fail to set them we won't notice and the system will behave unexpectedly. So simply get them as a dict, so it will fail fast:
```py
AWS_STORAGE_BUCKET_NAME = os.environ['BUCKET_NAME']
```

Reply:
`AWS_AUTO_CREATE_BUCKET` is set to `True` so the default bucket name will be created automatically.